lib/plugins/aws/package/compile/events/stream.js

```diff
@@ -9,6 +9,19 @@ class AwsCompileStreamEvents {
     this.provider = this.serverless.getProvider('aws');
 
     this.hooks = {
+      'initialize': () => {
+        if (
+          this.serverless.service.provider.name === 'aws' &&
+          this.serverless.service.provider.kinesis &&
+          this.serverless.service.provider.kinesis.consumerNamingMode &&
+          this.serverless.service.provider.kinesis.consumerNamingMode === 'serviceSpecific'
+        ) {
+          this.serverless._logDeprecation(
```

Comment:
We should document this in `deprecations.md` doc - please see the short guide about configuring deprecations: https://github.com/serverless/serverless/blob/master/CONTRIBUTING.md#configuring-deprecations

Reply:
Will do this